components/definitions/rewards.d.ts

```diff
@@ -175,7 +177,8 @@ declare namespace Rewards {
     NOT_VERIFIED = 0,
     CONNECTED = 1,
     UPHOLD_VERIFIED = 2,
-    BITFLYER_VERIFIED = 3
+    BITFLYER_VERIFIED = 3,
+    GEMINI_VERIFIED = 4
```

Comment:
sadly this also needs to go in `rewardsExtensions.d.ts`

Reply:
added: https://github.com/brave/brave-core/pull/8566/files#diff-2548406e37da614e6f5858694b3adf292d92dc032e07fb2e7a7c3ecd2a50b12fR35